hbase-server/src/test/java/org/apache/hadoop/hbase/wal/TestWALSplit.java

```diff
@@ -379,7 +379,7 @@ private void loop(final Writer writer) {
   }
 
   /**
-   * {@see https://issues.apache.org/jira/browse/HBASE-3020}
+   * @see "https://issues.apache.org/jira/browse/HBASE-3020"
```

Comment:
I think that you can remove the quotes

Reply:
If we remove the quotes it turns read and gives a message of cannot resolve symbol.

Reply:
Got it. Then it's fine.